Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -94,7 +95,7 @@ public bool IsLocalBackupExist
         {
             get
             {
-                if (mLocalBackupDic != null)
+                if (mLocalBackupDic != null && mBackupInProgress == false)
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5848147)